<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogWidget</class>
 <widget class="QWidget" name="LogWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>220</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>10</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Compass Log</string>
  </property>
  <property name="toolTip">
   <string>Save, print, or copy text</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_ToolBar">
     <item>
      <widget class="QToolButton" name="toolButton_text">
       <property name="text">
        <string> Text   </string>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextOnly</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
       <property name="arrowType">
        <enum>Qt::NoArrow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_logging">
       <property name="statusTip">
        <string>Select message logging options.</string>
       </property>
       <property name="text">
        <string>Logging   </string>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextOnly</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
       <property name="arrowType">
        <enum>Qt::NoArrow</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_help">
       <property name="text">
        <string>Help   </string>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextOnly</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>10</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="backgroundVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="action_Reset">
   <property name="icon">
    <iconset resource="compass.qrc">
     <normaloff>:/images/file-reset.png</normaloff>:/images/file-reset.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Reset</string>
   </property>
   <property name="toolTip">
    <string>Clear the current text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="compass.qrc">
     <normaloff>:/images/file-save.png</normaloff>:/images/file-save.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As</string>
   </property>
   <property name="toolTip">
    <string>Save current text into a file</string>
   </property>
  </action>
  <action name="action_Print">
   <property name="icon">
    <iconset resource="compass.qrc">
     <normaloff>:/images/file-print.png</normaloff>:/images/file-print.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Print</string>
   </property>
   <property name="toolTip">
    <string>Print current text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="icon">
    <iconset resource="compass.qrc">
     <normaloff>:/images/edit-select-all.png</normaloff>:/images/edit-select-all.png</iconset>
   </property>
   <property name="text">
    <string>Select &amp;All</string>
   </property>
   <property name="toolTip">
    <string>Select all current text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="icon">
    <iconset resource="compass.qrc">
     <normaloff>:/images/edit-copy.png</normaloff>:/images/edit-copy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy all selected text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionMessage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Message</string>
   </property>
   <property name="toolTip">
    <string>Log all standard messages</string>
   </property>
  </action>
  <action name="actionWarning">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Warning</string>
   </property>
   <property name="toolTip">
    <string>Log all Warning messages</string>
   </property>
  </action>
  <action name="actionRaw_Data">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Raw Data</string>
   </property>
   <property name="toolTip">
    <string>Log all Raw data (dump)</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Debug</string>
   </property>
   <property name="toolTip">
    <string>Log all debug messages</string>
   </property>
  </action>
  <action name="action_Help">
   <property name="icon">
    <iconset resource="compass.qrc">
     <normaloff>:/images/help-browser.png</normaloff>:/images/help-browser.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="compass.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionSelect_All</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Copy</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
